### PR TITLE
Add defines for old motor

### PR DESCRIPTION
**Pull request by AntonOstman**
Everything is best and tested

### CODE REVIEW

File: CM7/Core/Src/nav.c [hunk 1]
```diff
@@ -75,6 +76,20 @@ void NAV_Init(TIM_HandleTypeDef* motor_tick_itr,
...
   motors[1].dir               = 1;
+#else
+  motors[1].pwm_htim          = pwm_htim;
+  motors[1].ticks             = 0;
+  motors[1].speed             = 0.f;
+  motors[1].prev_tick         = 0;
+  motors[1].encoder_htim      = encoder2_htim;
+  motors[1].channel           = TIM_CHANNEL_2;
+  //motors[1].breakPinPort      = OLD_MOTOR2_BREAK_GPIO_Port;
+  //motors[1].breakPin          = OLD_MOTOR2_BREAK_Pin;
```
Comment (r0x57n) [inline CM7/Core/Src/nav.c:87]: commented code

**Review by r0x57n — CHANGES_REQUESTED** [review]
fix plz

File: CM7/Core/Src/nav.c (hunk 1 above)
Comment (AntonOstman) [reply CM7/Core/Src/nav.c:87]: should be so

**Review by r0x57n — APPROVED** [review]
NOiCE

**Pull request merged**